emails/views.py

```diff
@@ -194,6 +201,12 @@ def _sns_inbound_logic(topic_arn, message_type, json_body):
             extra={'json_body': json_body},
         )
         return _sns_notification(json_body)
+    
+    logger.error(
+        'SNS message type did not fall under the SNS inbound logic',
+        extra={'message_type': message_type}
+    )
+    return HttpResponse('Received SNS message with type not handled in inbound log', status=400)
```

Comment:
I think we should also surface this error on Sentry, if possible. Otherwise we can create a follow-up issue for getting the errors to show up Sentry.

Reply:
Sounds good, I'll see what I can do!